lib/codit/population/person.py

```diff
@@ -66,7 +64,6 @@ def isolating(self):
 
     def recover(self):
         self.infectious = False
-        self.immune = True
```

Comment:
I think this self.immune would become obsolete when we start to use self.immunities to record type of immunity.
Self.immunities is much more flexible which can also be used to record different type of vaccine's immunities' set later on.